src/utils/io/can.cpp

```diff
@@ -26,23 +26,23 @@
 #include <sys/socket.h>
 #include <net/if.h>
 
-#ifndef WIN
-#include <linux/can.h>
+#if LINUX
```

Comment:
Please define this when cross compiling. Furthermore, `#if <VAR>` checks the value is true (non-zero), while `#ifdef <VAR>` checks the variable exists. In other words, the first one requires `-DVAR=1` compiler flag, while the second requires only `-DVAR`.

Reply:
`-DVAR` [without anything specified](https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html#Preprocessor-Options) defaults to `1`, so the `#if` should still catch it.

Reply:
good to know